content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -445,6 +445,80 @@ In this subsection, perform the same steps as in the [Generate client and server
         `"""`
     {{< /text >}}
 
+## Configure end-user authentication on ingress gateway
+
+The Istio ingress gateway supports end-user authentication by setting up a JWT
+authentication policy in the `istio-ingressgateway` file. The ingress gateway
```

Comment:
I'm not sure this is technically accurate. Is setting things up in the `istio-ingressgateway` configuration file or for the pod? Or is it the configuration file for the `istio-ingressgateway` pod?

Reply:
technically istio-ingressgateway is a service and authN policy targets to ```service```, so updated to ```service``` instead of ```pod```.